cypress/support/pageBuilder/pbListPageBlocks.js

```diff
@@ -0,0 +1,35 @@
+import { GraphQLClient } from "graphql-request";
+
+Cypress.Commands.add("pbListPageBlocks", () => {
+    cy.login().then(user => {
+        const client = new GraphQLClient(Cypress.env("GRAPHQL_API_URL"), {
+            headers: {
+                authorization: `Bearer ${user.idToken.jwtToken}`
+            }
+        });
+
+        const query = `
+            query ListBlockCategories {
+                pageBuilder {
+                    listPageBlocks {
+                        data {
+                            id
+                        }
+                        error {
+                            code
+                            message
+                            __typename
+                        }
+                        __typename
+                    }
+                }
+            }
+        `;
+
+        return client.request(query).then(response => {
```

Comment:
To be more consistent with the rest of the code, let's have this command return an array of objects, not just IDs. For example: `cypress/support/pageBuilder/pbListPages.js:1`.

This will also enable us to potentially access other properties of returned block categories, not just their IDs.

Reply:
I can see this is still pending.